runtime/port/zos390/protect_helpers.c

```diff
@@ -46,9 +46,11 @@ protect_memory(struct J9PortLibrary *portLibrary, void *address, uintptr_t lengt
 
 	if ((flags & OMRPORT_PAGE_PROTECT_WRITE) == 0) {
 		/*omrtty_printf(portLibrary,"Calling _MPROT addr=%d length=%d\n",address,length);*/
+		extern intptr_t _MPROT(uintptr_t address, uintptr_t length);
```

Comment:
I understand that _MPROT and _MUNPROT is only called inside the function, it might be better for readability to have extern in the beginning after header declaration ? AFAIK, this one would not make any difference here.

Reply:
I'd characterize this as a matter of style: I have no objections to moving the declaration earlier as suggested.